tide_core.info.yml

```diff
@@ -70,6 +68,8 @@ dependencies:
   - viewsreference:viewsreference
   - wysiwyg_template:wysiwyg_template
   - wysiwyg_template:wysiwyg_template_core
+  - drupal:scheduled_transitions
```

Comment:
Why do these start with `drupal` and aren't of the form `scheduled_transitions:scheduled_transitions`?

Reply:
changed.